hoomd/md/pair/pair.py

```diff
@@ -1602,10 +1602,10 @@ class OPP(Pair):
     coefficients. The coefficients must be set per unique pair of particle
     types.
 
-    The potential comes from Marek Mihalkovič and C. L. Henley 2012
-    `paper link`_.
+    The potential comes from `Marek Mihalkovič and C. L. Henley 2012`_.
 
-    .. _paper link: https://dx.doi.org/10.1103/PhysRevB.85.092102
```

Comment:
multiple uses of `paper link` are fine you just need anonymous links which use two underscores instead of one.

Reply:
Yes, but descriptive lines are more accessible: https://usability.yale.edu/web-accessibility/articles/links

Reply:
Thanks for the link! I did not think of this. I made this change elsewhere for the line constraint, but we should revert those changes in another PR.